tasks/snapshot.yml

```diff
@@ -2,7 +2,8 @@ version: '3'
 
 vars:
   PREPARE_DIRECTORY: |
-    rm -rf .git
+    find . -type d -name ".git" | xargs rm -rf
```

Comment:
I think we probably need to use `-print0` for find and `-0` for xargs, in case there's any spaces in file or directory names?

Reply:
fixed :+1: